### ISSUE TITLE
Install apt repo for newer versions via package

### ISSUE THREAD

**Issue by colby** [copious-cookbooks/nginx#3]
Add ubuntu GPG key to Apt
Add Apt `sources.d` file for http://nginx.org/packages/ubuntu/dists/
Apt update
Apt install Nginx version

**Comment by colby** [comment]
Current versions supported via package.
```bash
vagrant@default-ubuntu-1404:~$ apt-cache policy nginx
nginx:
  Installed: 1.4.6-1ubuntu3.5
  Candidate: 1.4.6-1ubuntu3.5
  Version table:
 *** 1.4.6-1ubuntu3.5 0
        500 http://us.archive.ubuntu.com/ubuntu/ trusty-updates/main amd64 Packages
        500 http://security.ubuntu.com/ubuntu/ trusty-security/main amd64 Packages
        100 /var/lib/dpkg/status
     1.4.6-1ubuntu3 0
        500 http://us.archive.ubuntu.com/ubuntu/ trusty/main amd64 Packages
```

```bash
vagrant@default-ubuntu-1604:~$ apt-cache policy nginx
nginx:
  Installed: 1.10.0-0ubuntu0.16.04.2
  Candidate: 1.10.0-0ubuntu0.16.04.2
  Version table:
 *** 1.10.0-0ubuntu0.16.04.2 500
        500 http://us.archive.ubuntu.com/ubuntu xenial-updates/main amd64 Packages
        500 http://us.archive.ubuntu.com/ubuntu xenial-updates/main i386 Packages
        500 http://security.ubuntu.com/ubuntu xenial-security/main amd64 Packages
        500 http://security.ubuntu.com/ubuntu xenial-security/main i386 Packages
        100 /var/lib/dpkg/status
     1.9.15-0ubuntu1 500
        500 http://us.archive.ubuntu.com/ubuntu xenial/main amd64 Packages
        500 http://us.archive.ubuntu.com/ubuntu xenial/main i386 Packages
```

**Comment by colby** [comment]
Fixed in dc3b70e0ada8f49b71965a0961176b68d11e9aa8
